theme/css/colors.css

```diff
@@ -0,0 +1,28 @@
+/* Elements-related colors */
+.c-sol {
```

Comment:
Do these colors look good with both the default and the dark mdbook themes?

Reply:
Is this good enough?
I've adjusted the d-dark color, it was too dark to be read properly on dark themes.
![elem_colors](https://github.com/shenef/Boktai-Speedrunning/assets/17726521/6e483e0b-c928-4e19-853b-2b54d238f2b0)